packages/salesforcedx-vscode-soql/src/queryDataView/queryDataFileService.ts

```diff
@@ -85,7 +85,7 @@ export class QueryDataFileService {
   private showFileInExplorer(targetPath: string) {
     vscode.commands.executeCommand(
       'revealInExplorer',
-      vscode.Uri.parse(targetPath)
+      vscode.Uri.file(targetPath)
```

Comment:
DOH! That was silly of me, we use file everywhere else.

I wonder why this works on macOS regardless?

Reply:
I think it works on MacOS because of the format of the string being passed to the parse method. It could be that on Windows the `targetPath` does not include the scheme or has some characters that are not correctly handled. You can add breakpoints in the code and look at the differences in `targetPath` across OS. Here's more info about the differences in methods https://code.visualstudio.com/api/references/vscode-api#Uri.

Reply:
Thanks for the link, they example they have in the docs helps to see.